### PR TITLE
[sc-6075] Provide chat as an independent web component 

### PR DESCRIPTION
**Pull request by operramon**

### CODE REVIEW

**Review by mpellerin42 — APPROVED** [review]

File: libs/search-widget/src/components/answer/Chat.scss [hunk 1]
```diff
@@ -1,40 +1,60 @@
...
+      right: var(--rhythm-1);
+      top: var(--rhythm-1);
+    }
   }
 
   .chat-container {
     --chat-padding: var(--rhythm-1_5);
-    --height-input: var(--rhythm-6);
-    --max-width-container: calc(100vw - var(--main-padding-side) * 2);
+    --height-container: var(--custom-height-container, 500px);
```
[the remark this answers is not in the document]
Comment (ebrehault) [reply libs/search-widget/src/components/answer/Chat.scss:29]: that was my suggestion, but yes indeed 480 would be better

File: tools/build-widgets.sh [hunk 2]
```diff
@@ -11,5 +11,11 @@ vite build -c=libs/search-widget/vite.config.js -- viewer-widget nuclia-viewer-w
 mv dist/libs/viewer-widget/style.css dist/libs/search-widget/nuclia-viewer-widget.css
 cp dist/libs/viewer-widget/* dist/libs/search-widget
 
+echo "Build chat widget"
+vite build -c=libs/search-widget/vite.config.js -- chat-widget nuclia-chat-widget
+mv dist/libs/chat-widget/style.css dist/libs/search-widget/nuclia-chat-widget.css
```
Comment (ebrehault) [inline tools/build-widgets.sh:16]: That's correct, but we must also change our `.github/workflows/deploy.yml` so it publishes this file to the CDN.
Visibly we forgot to do so for the viewer style, I guess a command like
```
gsutil copy dist/libs/search-widget/*.css gs://$CDN_STORAGE
```
would fix it for all the existing and future widgets.

**Review by ebrehault — APPROVED** [review]

File: libs/search-widget/src/components/answer/Chat.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (operramon) [reply libs/search-widget/src/components/answer/Chat.scss]: Fixed

File: tools/build-widgets.sh (hunk 2 above)
Comment (operramon) [reply tools/build-widgets.sh:16]: Fixed

**Pull request merged**